apps/service_profile/lib/profile/feed/flow.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Profile.Feed.Flow do
+  use Flow
+  use Properties, otp_app: :service_profile
+  require Logger
+
+  alias Profile.Feed.Flow.Storage
+
+  getter(:window_limit, default: 5)
+  getter(:window_unit, default: :minute)
+
+  @type init_opts :: [
+          name: GenServer.name(),
+          dataset_id: String.t(),
+          subset_id: String.t(),
+          from_specs: list(Supervisor.child_spec()),
+          into_specs: list(Supervisor.child_spec()),
+          reducers: list(Profile.Reducer.t())
+        ]
+
+  @spec start_link(init_opts) :: GenServer.on_start()
+  def start_link(opts) do
+    Logger.debug(fn -> "#{__MODULE__}(#{inspect(self())}): init with #{inspect(opts)}" end)
+    flow_opts = Keyword.take(opts, [:name])
+
+    dataset_id = Keyword.fetch!(opts, :dataset_id)
+    subset_id = Keyword.fetch!(opts, :subset_id)
+    from_specs = Keyword.fetch!(opts, :from_specs)
+    into_specs = Keyword.fetch!(opts, :into_specs) |> Enum.map(fn consumer -> {consumer, []} end)
+    reducers = Keyword.fetch!(opts, :reducers)
+
+    window = Flow.Window.periodic(window_limit(), window_unit())
+
+    {:ok, stats} = Profile.Feed.Store.get_stats(dataset_id, subset_id)
+    reducers = Enum.map(reducers, &Profile.Reducer.init(&1, stats))
+    {:ok, storage} = Storage.start_link(reducers: reducers)
```

Comment:
This name seems to imply the Store modules throughout the umbrella that refer to Brook; I think it makes it less clear with such similar names.

Reply:
I changed it to Profile.Feed.Flow.State.  Does that sound better?

Reply:
Yup